src/main/java/JobProcessorApplication.java

```diff
@@ -13,23 +16,20 @@ public static void main(String args[])
 
         try {
             LOG.info("Job started----");
-            LOG.info("Slot----"+args[0]);
             JobProcessorApplication application = new JobProcessorApplication();
-            if(args[0].trim().equalsIgnoreCase("PRODUCTION")) {
-                LOG.info("App slot is supported----");
-                S2STokenGeneration s2STokenGeneration = new S2STokenGeneration();
-                String s2sToken = s2STokenGeneration.generateOTP(args[1], args[2], args[3]);
-                LOG.info("s2sToken is generated");
-                application.getJobProcessor(args[4], args[5], s2sToken);
-            }
-            else
-            {
-                throw new IllegalArgumentException("Unsupported app slot to run this application"+args[0]);
-            }
+            String clientId = new String(Files.readAllBytes(Paths.get("/mnt/secrets/ccpay/gateway-s2s-client-id")));
```

Comment:
/mnt/secrets/ccpay should really be extracted to an env var, (this could be a default if you wanted)